### ISSUE TITLE
project progress

### ISSUE THREAD

**Issue by bilkaPek** [VatConnect/FSX#9]
Hi
you have abandoned the project ?

thank you

**Comment by cfcoon** [comment]
No, I've been working on the server interface while I wait for Dovetail Games to release their SDK to Flight Sim World. I was going to focus on that because the new SWIFT pilot client by a VATSIM group is getting closer to completion and will cover FSX, P3D and XPlane, (they say they will make it open-source). 

If FSW doesn't release their SDK pretty soon, I may switch over to P3Dv4 because like FSW it is 64 bit and uses DirectX11. I don't know if the new SWIFT client will support full-screen DirectX, but VatConnect does so it may be useful to some.

**Comment by bilkaPek** [comment]
Hi cfcoon
Sorry for the late reply I understand your expectation for swift, I also expect it :), but since he says they will publish it.
I recently acquired the FSD server that I have set up on my server for private sessions and I would like to know if in the current state your project would be functional and could be used because I currently use a solution that does not Really like me.
If it is the case you can make a build to make tests.
sorry for my English
greetings.

SAM

**Comment by cfcoon** [comment]
Right now it only connects to VATSIM servers, which are a modification to the FSD server (there is a different login procedure, and it looks for certain servers that announce the IP's for the VATSIM servers). 

You would have to re-write the ServerInterface project to respond to the messages in packets.h, then translate to connect to your FSD server. That's not very easy to do. Also, the model matching is not written yet, so it only shows one type of aircraft. 

I think you should look at FSInn, which is a pilot client that can connect to a standard FSD server. There are some problems to set it up, but on google you can find instructions to try to make it work. 

Good luck!
